RuleDocumentation/UseApprovedVerbs.md

```diff
@@ -8,6 +8,8 @@ All CMDLets must used approved verbs.
 
 Approved verbs can be found by running the command `Get-Verb`.
 
+Additional documentation on approved verbs can be found in the microsoft docs page [Approved Verbs for PowerShell Commands](https://docs.microsoft.com/en-us/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands). If you find the verb you are using is unapproved, try searching the page for the approved equivalent. For example, if you search in the documentation for `Read`, `Open`, or `Search` you will find that the approved verb for those situations is `Get`.
```

Comment:
Thanks. Looks good to me. Just a minor suggestion to take out the `\en-us` part of the link so that the page gets displayed in the user's locale.
```suggestion
Additional documentation on approved verbs can be found in the microsoft docs page [Approved Verbs for PowerShell Commands](https://docs.microsoft.com/en-us/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands). If you find the verb you are using is unapproved, try searching the page for the approved equivalent. For example, if you search in the documentation for `Read`, `Open`, or `Search` you will find that the approved verb for those situations is `Get`.
```

Reply:
Thanks for the suggestion. I have removed that. Do I need to squash both changes into one commit before the pull request is completed?

Reply:
You're welcome. No need for squashing, when the PR gets merged all commits will get squashed automatically by GitHub and the final commit message will be the PR title and the description